### PR TITLE
implemented search and picture storage

### PR DESCRIPTION
**Pull request by olehsalo**
We have implemented search logic for dishes and also front end is now able to show real data from sessionstorage and images. Also we have upgraded information in the database.

### CODE REVIEW

**Comment by olehsalo** [comment]
don't forget to write pip install django-cors-headers before starting server

File: front_end/src/App.jsx [hunk 1]
```diff
@@ -4,8 +4,10 @@ import { withNamespaces } from 'react-i18next';
 import Main from './containers/Main/Main.jsx';
 import { storeDataToSessionStorage } from './api/session_storage.js';
 
+
 function App({ t }) {
-    storeDataToSessionStorage();   
+
+    storeDataToSessionStorage();
```
Comment (rkoruk) [inline front_end/src/App.jsx:10]: let's fixed this, change App to class or use https://reactjs.org/docs/hooks-effect.html

File: front_end/src/components/dish.jsx [hunk 2]
```diff
@@ -1,22 +1,26 @@
...
-       
+
+
 	return(
                 <li className="list-group-item">
                 <div className="card mb-3 recipeCard">
                 <div className="row no-gutters">
                     <div className="col-md-4">
-                        <img src='...' className="card-img" alt={t("imgAlt")}></img>
+                        <img src={require(`../images/${this.props.name}.png`)} className="card-img" alt={t("imgAlt")}/>
```
Comment (rkoruk) [inline front_end/src/components/dish.jsx:16]: generally ok, but move to function or variable require(`../images/${this.props.name}.png`)

File: front_end/src/containers/Main/Main.jsx [hunk 3]
```diff
@@ -43,13 +40,13 @@ class Main extends React.Component {
                     </div>
                     <div className="col-md-7">
                         <ul className="list-group dishes" id="dishList">
-	     		    {this.state.dishList.map(item => (<Dish/>))}
+	     		    {this.state.dishList.map(item => (<Dish name = {item.dish_name} desc = {item.dish_description}/>))}
```
Comment (rkoruk) [inline front_end/src/containers/Main/Main.jsx:43]: props passing should be written the same way as general properties without spaces in between =

File: front_end/src/containers/Searchbar/Searchbar.jsx [hunk 4]
```diff
@@ -5,14 +5,26 @@ import { withNamespaces } from 'react-i18next';
...
 
+
+
 	state = {
 		searchResult: []
 	}
-	
-	componentDidMount(){
+    handleCatChange(e) {
+	    var obj;
```
Comment (rkoruk) [inline front_end/src/containers/Searchbar/Searchbar.jsx:14]: fix indentation, also use `let`, also also, should be more meaningful name than `obj`.

few optimizations points:

move this part into the variable `JSON.parse(sessionStorage.getItem('dish')`

and in for...in loop use it.

let i should be renamed to `item` or something like that

and e renamed to event

File: front_end/src/containers/Searchbar/Searchbar.jsx [hunk 5]
```diff
@@ -39,9 +52,10 @@ class Searchbar extends React.Component {
                     <input type="radio" name="options"></input>
                 </label>
             </div>
-            <form action="#" className="form-inline mx-auto">
-                <select className="selectpicker searchbar " multiple data-live-search="true" data-style="btn-outline-warning" data-size="5"  >
+            <form action="#" className="form-inline mx-auto" placeholder="DropDown" onChange={this.handleCatChange.bind(this)}>
```
Comment (rkoruk) [inline front_end/src/containers/Searchbar/Searchbar.jsx:55]: .bind(this) with React 16, well stuff happens 😆😆

**Pull request merged**